config/configuration_file.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/sonatype-nexus-community/go-sona-types/ossindex/types"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type ChequeConfig struct {
+	ossiConfig OSSIConfig
```

Comment:
Starting something with lowercase makes it inaccessible outside of the package (Golangs version of public private). I am pretty sure you want it public, so I'd do `OSSIndexConfig` and `IQConfig`?

Reply:
Ok cool

Reply:
done!